app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -0,0 +1,43 @@
+$thumbnail-width: 99px;
+$thumbnail-height: 140px;
+$thumbnail-border: 5px;
+$thumbnail-border-colour: rgba(11, 12, 12, .1);
+$thumbnail-shadow-colour: rgba(11, 12, 12, .4);
+
+.gem-c-attachment {
+  @include govuk-font(19);
+  @include govuk-clearfix;
+}
+
+.gem-c-attachment__thumbnail {
+  float: left;
+  margin-bottom: govuk-spacing(3);
+  margin-right: govuk-spacing(5);
+  padding: $thumbnail-border;
+  position: relative;
+}
+
+.gem-c-attachment__thumbnail-image {
+  display: block;
+  width: $thumbnail-width;
+  max-width: 100%;
+  height: $thumbnail-height;
+  outline: $thumbnail-border solid $thumbnail-border-colour;
+  background: govuk-colour("white");
+  box-shadow: 0 2px 2px $thumbnail-shadow-colour;
+}
+
+.gem-c-attachment__title {
+  @include govuk-font($size: 27);
+  margin: 0 0 govuk-spacing(3);
+}
+
+.gem-c-attachment__metadata {
+  @include govuk-font($size: 14);
```

Comment:
We probably don't want to mess with the design too much but on mobile this shrinks to 12px which is pretty small, and there's loads of space available. Maybe increase this line to size 16?

Reply:
Checked this with our designer and probably the best option here is to use 14px for both desktop and mobile. What do you think about this @andysellick?

We can achieve this by explicitly declaring the font-size for mobile.
```
.gem-c-attachment__metadata {
    @include govuk-font($size: 14); 

    @include govuk-media-query($until: tablet) { 
      font-size: 14px;
    }
}
```